habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -721,13 +724,13 @@ def step(self, action: Union[str, int]) -> Observations:
 
     def maybe_update_robot(self):
         """
-        Calls the update robots method on the robot manager if the
+        Calls the update agents method on the robot manager if the
         `update_robot` configuration is set to True. Among other
         things, this will set the robot's sensors' positions to their new
         positions.
         """
         if self._update_robot:
-            self.robots_mgr.update_robots()
+            self.agents_mgr.update_agents()
```

Comment:
what if we want to update human agent, will it still be the same as the one for robot?

Reply:
Yes, as long as the human has grasp managers, it should update them the same way.